scripts/buildkite/release/release-template.md

```diff
@@ -0,0 +1,43 @@
+## Node Compatibility
+
+Compatible with [`cardano-node@$NODE_TAG`](https://github.com/input-output-hk/cardano-node/releases/tag/$NODE_TAG).
+
+## Docker Image
+
+FIX_ME !
```

Comment:
What to fix here ?

Reply:
The link. It's one of the still automatable things. For now we  have do it by hand going to dockerhub and search